.flake8

```diff
@@ -1,4 +1,4 @@
 [flake8]
 max-line-length = 120
-ignore = E305,E402,E721,E741,F401,F403,F405,F999,W503,W504
-exclude = build,docs/source,_ext,third_party
+ignore = E305,E402,E721,E741,F401,F403,F405,W503,W504,F821,F841,F999
+exclude = build,docs/source,_ext,third_party,docs/src,venv,torch/lib/gloo,torch/lib/pybind11,torch/lib/nanopb
```

Comment:
Let's remove unused ones. Only `build,docs/source,third_party` are relevant and the others are originated from PyTorch core.